src/certifier.mak

```diff
@@ -92,7 +90,9 @@ CERTIFIER_LIB = certifier.a
 LIBCERTIFIER         = lib$(SWIG_CERT_INTERFACE)
 CERTIFIER_SHARED_LIB = $(LIBCERTIFIER).so
 
-all:	$(CL)/$(CERTIFIER_LIB)
+# Include SEV-SNP code-fragments, too, when building .a
+all: ENABLE_SEV=1
+all: $(CL)/$(CERTIFIER_LIB)
```

Comment:
With L94, we will still continue to compile-in code under `#if SEV_SNP` while building the static library.

We just will not pull in that code while building the shared library, which is being used just for pytests now.

Once the whole release model is better understood, we may have to revisit these Makefile-rules, but this arrangement seems acceptable to me (for now).

Reply:
With @yelvmw : remove this change, so the SEV_SNP code is included in shared libs, too.

Reply:
@yelvmw -- I have updated this change set as we discussed. Now SEV_SNP code chunks will be compiled with shared libraries, also.

I am ready to push this to `/main` once tests pass.